1.intro/chapter3/customize/notebooks-customize.ipynb

```diff
@@ -0,0 +1,1068 @@
CvRYsWDBgwgG+//Tbb4wiXP9bO8t3MjSu9pjxx4gSdOnWiXbt2fPHFFw7L6Vjnj+JyzVtUFKfrWJFCV9hJYArT5ZLcps9C/vnnn1t+fn5WfHy8ZVkm0V5ISIjDev379y/0RHutWrWyRo4cmfo6JSXFqlSpUpFO+EU2SW7ffffd1GmRkZFZJrlcv3596jILFy4s8CSXdrvdGjFihBUUFGTt2bMn0/yLycFmzpyZOm3Xrl1ZJge71PetMHXq1MkaOHCgU+1LVFSUtXXrVodHixYtrPvuu8/aunWrU+1LVqKjo61SpUpZH374odPtS//+/TMluR09erTVtm1by7Kc6+//onHjxlkVKlSwkpKSUqc52+eS3y5+humT3CYmJlqBgYGp1VSc+dxblBw+fNjh/76FCxdagDVz5kzr6NGjlmWlJV5NTExMXe+5557LlHj11ltvddh227ZtlXg1g2PHjlm1a9e27rnnHis5OTnTfB3r/OOM17xFzbVwHStS0K7JAMvhw4etTZs2Wa+88orl6+trbdq0ydq0aZMVHR1tWVZaubGuXbtamzdvthYsWGCVK1cuy1KRTz31lLVz507r008/LRKlIqdPn255enpaU6ZMsXbs2GE9/PDDVkBAgENm76IgOjo69bgD1nvvvWdt2rTJOnz4sGVZpkxrQECA9dtvv1lbtmyx7rjjjizLtDZt2tRas2aN9e+//1q1a9cu8DKtw4cPt/z9/a2lS5c6lGuNi4tLXWbYsGFWcHCwtXjxYmv9+vVW27ZtU28mLStn37eC8uyzz1rLli2zDh48aG3ZssV69tlnLZvNZv35559Oty8Zpa8iZFnOtS9jx461li5dah08eNBasWKF1aVLF6ts2bJWeHi40+3L2rVrLTc3N+uNN96w9u7da/3444+Wj4+P9cMPP6Qu4yx//5ZlLuiDg4OtZ555JtM8Z/pcCsKoUaOsSpUqWQsXLrR27dplDRkyxAoMDLTOnj1rWZZzn3uLsoMHD2aqIhQREWGVL1/euv/++61t27ZZ06dPt3x8fDKVDnZzc7Peffdda+fOnda4ceNUOjiDY8eOWbVq1bI6d+5sHTt2zOE64CId6/zjLNe8RVlxu44VKQquyQDLwIEDLSDTY8mSJanLHDp0yOrRo4fl7e1tlS1b1ho7dqzDr5OWZVlLliyxmjRpYnl4eFg1atSwJk+eXLA7ko2PP/7YCg4Otjw8PKxWrVpZq1evLuwmZbJkyZIsP4OBAwdalmUi6i+++KJVvnx5y9PT0+rcubO1e/duh22cOXPG6t+/v+Xr62v5+flZDz74YGqQrKBktQ+Aw3fh/Pnz1qOPPmqVKlXK8vHxsXr37u1w8WVZOfu+FYTBgwdbVatWtTw8PKxy5cpZnTt3Tg2uWJZz7UtGGQMszrQv/fr1sypWrGh5eHhYlSpVsvr162ft27cvdb4z7YtlWdYff/xhhYSEWJ6enla9evWsL774wmG+s/z9W5aV2jsgY/ssy/k+l/yWmJhojR071goMDLRKlixpdenSxdq2bZvDMs587i2qsgqwWJZl/ffff1b79u0tT09Pq1KlStZbb72Vad0ZM2ZYderUsTw8PKyGDRtac+fOLaBWO4fJkydnex2Qno51/nGGa96irLhdx4oUBTbLsqz8Gn4kIiIiIiIiInItKDZVhERERERERERECosCLCIiIiIiIiIiuaQAi4iIiIiIiIhILinAIiIiIiIiIiKSSwqwiIiIiIiIiIjkkgIsIiIiIiIiIiK5pACLiIiIiIiIiEguKcAiIiIiIiIiIpJLCrCIyFVbunQpNpuNiIiIwm6KiIiIU9M5VUTE+dksy7IKuxEi4hw6duxIkyZN+OCDDwBITEzk7NmzlC9fHpvNVriNExERcSI6p4qIFD9uhd0AEXFeHh4eVKhQobCbISIi4vR0ThURcX4aIiQiOTJo0CCWLVvGhx9+iM1mw2azMWXKFIfuzFOmTCEgIIA5c+ZQt25dfHx86Nu3L3FxcXz77bdUq1aNUqVK8fjjj5OSkpK67YSEBJ588kkqVapEiRIlaN26NUuXLi2cHRUREclnOqeKiBRP6sEiIjny4YcfsmfPHkJCQnj11VcB2L59e6bl4uLi+Oijj5g+fTrR0dHceeed9O7dm4CAAObNm8eBAwfo06cP119/Pf369QNg5MiR7Nixg+nTpxMUFMSsWbPo3r07W7dupXbt2gW6nyIiIvlN51QRkeJJARYRyRF/f388PDzw8fFJ7cK8a9euTMslJSUxceJEatasCUDfvn35/vvvCQsLw9fXlwYNGtCpUyeWLFlCv379OHLkCJMnT+bIkSMEBQUB8OSTT7JgwQImT57Mm2++WXA7KSIiUgB0ThURKZ4UYBGRPOXj45N6IQhQvnx5qlWrhq+vr8O08PBwALZu3UpKSgp16tRx2E5CQgJlypQpmEaLiIgUQTqniog4FwVYRCRPubu7O7y22WxZTrPb7QDExMTg6urKhg0bcHV1dVgu/QWkiIjItUbnVBER56IAi4jkmIeHh0MivbzQtGlTUlJSCA8Pp0OHDnm6bRERkaJK51QRkeJHVYREJMeqVavGmjVrOHToEKdPn079xSw36tSpw4ABA3jggQf49ddfOXjwIGvXrmX8+PHMnTs3D1otIiJS9OicKiJS/CjAIiI59uSTT+Lq6kqDBg0oV64cR44cyZPtTp48mQceeICxY8dSt25devXqxbp16wgODs6T7YuIiBQ1OqeKiBQ/NsuyrMJuhIiIiIiIiIiIM1MPFhERERERERGRXFKARUREREREREQklxRgERERERERERHJJQVYRERERERERERySQEWEREREREREZFcUoBFRERERERERCSXFGAREREREREREcklBVhERERERERERHJJARYRERERERERkVxSgEVEREREREREJJcUYBERERERERERySUFWEREREREREREcun/AZzWSLfHkvJ5AAAAAElFTkSuQmCC",
```

Comment:
this image is wonderful! didn't you find out graphics rendering with matplotlib and such similar libs have problems in default image shipped by redhat? do you want to call out or highlight these things here in this section? is there something data scientists should do to work with free open source as well as popular commercial plotting libs as part of their regular day jobs??